src/utils/hooks/useTriggerProcessing.tsx

```diff
@@ -1,16 +1,41 @@
 import { useState, useEffect } from 'react'
 import { useGetTriggersQuery } from '../../utils/generated/graphql'
 
-type triggerTypes = 'applicationTrigger' | 'reviewTrigger'
+type TriggerType = 'applicationTrigger' | 'reviewAssignmentTrigger' | 'reviewTrigger'
 
-const useTriggerProcessing = (props: { serialNumber: string; trigger: triggerTypes }) => {
-  const { serialNumber, trigger } = props
+interface TrigerQueryProps {
+  serialNumber?: string
+  reviewAssignmentId?: number
+  reviewId?: number
+  triggerType?: TriggerType
+}
+
+const useTriggerProcessing = (props: TrigerQueryProps) => {
+  const { serialNumber, reviewAssignmentId, reviewId } = props
```

Comment:
```suggestion
const useTriggerProcessing = ({ serialNumber, reviewAssignmentId, reviewId, triggerType }: TrigerQueryProps) => {
```
Than also change elsewhere using `props.triggerType`